src/node_file.cc

```diff
@@ -3385,7 +3429,8 @@ static void CreatePerIsolateProperties(IsolateData* isolate_data,
   SetMethod(isolate, target, "accessSync", AccessSync);
   SetMethod(isolate, target, "close", Close);
   SetMethod(isolate, target, "closeSync", CloseSync);
-  SetMethod(isolate, target, "existsSync", ExistsSync);
+  SetFastMethodNoSideEffect(
```

Comment:
I am still not sure if it's safe to declare this to be free of side effects (see also https://github.com/nodejs/node/pull/49748#pullrequestreview-1639236167). `statSync` etc. are also not marked as free of side effects.

Reply:
Ah, @CanadaHonk added a better reference while I was typing 😄